docs/proposals/20201123-tracing-ingestion-api-and-opentelemetry-collector.md

```diff
@@ -0,0 +1,86 @@
+# Traces ingestion API and OpenTelemetry collector
+
+* **Owners:**:
+    * `@pavolloffay`
+
+* **Related Tickets:**
+    * [observatorium/issues/442](https://github.com/observatorium/observatorium/issues/442)
+
+* **Other docs:**
+
+## Table of Contents
+
+- [TLDR](#tldr)
+- [Why](#why)
+- [Goals](#goals)
+- [Non-Goals](#non-goals)
+- [How](#how)
+  * [Expose tracing ingestion API in the Observatorium API service](#expose-tracing-ingestion-api-in-the-observatorium-api-service)
+    + [Multitenancy](#multitenancy)
+  * [Add deployments manifests for OpenTelemetry collector](#add-deployments-manifests-for-opentelemetry-collector)
+- [Action Plan](#action-plan)
+- [References](#references)
+
+<small>
+
+<i>
+
+<a href="http://ecotrust-canada.github.io/markdown-toc/">
+Table of contents generated with markdown-toc
+</a>
+
+</i>
+
+</small>
+
+## TLDR
+
+This document proposes to expose tracing ingestion API  backed by the OpenTelemetry collector. This is a first step to support ingestion and storage for tracing data. The OpenTelemetry collector gives flexibility to forward tracing data to variety of tracing platforms, both on-prem or SaaS - Jaeger, Zipkin, Tempo, SigNoz or DataDog, Dynatrace etc.
+
+## Why
+
+At the moment Observatorium does not expose API to ingest distributed tracing data.
+
+## Goals
+
+* Expose API to ingest tracing data.
+* Deploy a component that receives tracing data and is able to forward it to tracing platforms with storage.
```

Comment:
>  For example, should tenants be able to configure what kind of exporters or processor should be used in the Collector? 

I don't think so. Processors are used for data manipulation/mutation and this can be done before in the customer's cluster before reporting it to the platform/observatorium.

> Is that goal for this proposal or perhaps will be goal for a future proposal?

I don't see this use case at the moment.

Reply:
Ok, so if I understand correctly, for example tenant won't be able to configure how or where the data is exported, but this will be governed by the operator, correct? I'm trying to clarify this with regards to what's stated in multitenancy, i.e. that collector could route to different exporters based on tenants. But my assumption is that after ingestion, only operator will decide how data is exported and the user will have no say in it (maybe could be worth specifying in non-goals).